content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -140,6 +140,38 @@ Exit your shell:
 exit
 ```
 
+## Configure volume permission and ownership change policy for Pods
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+Kubernetes by default recursively changes ownership and permission of volume to match `fsGroup` specified in pod's
+`securityContext` when volume is mounted. Changing ownership and permissions of a large volume can be time consuming
+operation and that is where `fsGroupChangePolicy` field in `securityContext` comes in.
```

Comment:
nit / my suggestion (only revise if something bigger also needs fixing, or if you want to):
```suggestion
By default, Kubernetes recursively changes ownership and permissions for the contents of each
volume to match the `fsGroup` specified in a Pod's `securityContext` when that volume is
mounted.
For large volumes, checking and changing ownership and permissions can take a lot of time,
slowing Pod startup. You can use the `fsGroupChangePolicy` field inside a `securityContext`
to control the way that Kubernetes checks and manages ownership and permissions
for a volume.
```
```